### PR TITLE
fiona/ end trading platform and trade-type pages url with a slash

### PR DESCRIPTION
**Pull request by fiona-deriv**
Changes:

- end trade card pages with a slash

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/other-platforms.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                     }
                     title={<Localize translate_text="DMT5" />}
                     onClick={onClick}
-                    to={is_ppc_redirect ? '/landing/dmt5' : '/dmt5'}
+                    to={!is_ppc_redirect ? '/dmt5/' : '/landing/dmt5/'}
```
Comment (sean-binary) [inline src/components/custom/other-platforms.js:267]:
```suggestion
                    to={is_ppc_redirect ? '/landing/dmt5/' : '/dmt5/'}
```

Comment (sean-binary) [reply src/components/custom/other-platforms.js:267]: Let's keep it this way for better clarity 😄

Comment (yashim-deriv) [reply src/components/custom/other-platforms.js:267]: i concurrr

**Comment by suthesan-deriv** [comment]
Nice work fiona!...just to add on there are a lot of other pages that also has links to these paths. We should change all of them too to end with a slash. You can easily do global search for "/margin" or "/dbot" to know all the places that needs to be changed ya 👌

File: src/components/custom/other-platforms.js (hunk 1 above)
Comment (fiona-deriv) [reply src/components/custom/other-platforms.js:267]: Okay I'll revert that

File: src/components/custom/other-platforms.js [hunk 2]
```diff
@@ -264,7 +264,7 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                     }
                     title={<Localize translate_text="DMT5" />}
                     onClick={onClick}
-                    to={is_ppc_redirect ? '/landing/dmt5' : '/dmt5'}
+                    to={!is_ppc_redirect ? '/landing/dmt5/' : '/dmt5/'}
```
Comment (sean-binary) [inline src/components/custom/other-platforms.js:267]:
```suggestion
                    to={is_ppc_redirect ? '/landing/dmt5/' : '/dmt5/'}
```

Comment (sean-binary) [reply src/components/custom/other-platforms.js:267]: Think you missed out on the `!`

**Review by yashim-deriv — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**